go/nbs/mem_table.go

```diff
@@ -69,11 +72,12 @@ func (mt *memTable) get(h addr) []byte {
 	return mt.chunks[h]
 }
 
-func (mt *memTable) getMany(reqs []getRecord, wg *sync.WaitGroup) (remaining bool) {
-	for i, r := range reqs {
+func (mt *memTable) getMany(reqs []getRecord, foundChunks chan *chunks.Chunk, wg *sync.WaitGroup) (remaining bool) {
```

Comment:
Yeah. I see what you are saying. I think I'm going to open a bug for this so we can think about it. As you point out, fixing it would involve basically making an internal notion of chunk, which seems to suggest

-It's reasonable for a store to depend on Chunk
-It's probably better to move the other stores away from the `chunks` package.

For instance, we could move the chunk specific stuff together with `hash` (since it's pretty annoying that that name collides with the stdlib) and put all ChunkStore implementations in `store` or something.